jsk_2015_05_baxter_apc/launch/segmentation_in_bin_atof.launch

```diff
@@ -11,14 +11,10 @@
         pkg="nodelet" type="nodelet"
         args="load jsk_pcl/AttentionClipper $(arg MANAGER)">
     <remap from="~input/points" to="$(arg INPUT_CLOUD)" />
+    <rosparam file="$(find jsk_apc2016_common)/config/bin_shelf_bottom.yaml" command="load"/>
```

Comment:
it is not good for 2015 to depend on 2016.
you should use arg of roslaunch to pass the yaml file.
 Add a line note